jlm/hls/backend/rhls2firrtl/verilator-harness-hls.cpp

```diff
@@ -40,8 +40,8 @@ VerilatorHarnessHLS::get_text(llvm::RvsdgModule & rm)
          "#include \"verilated_vcd_c.h\"\n"
          "#endif\n"
          "#include \"V"
-      << file_name << ".h\"\n"
-      << "#define V_NAME V" << file_name << "\n"
+      << GetIncludeFileName() << ".h\"\n"
```

Comment:
Throw the `.h` out, i.e., `<< GetIncludeFilename().to_str() << "\"\n"`

Reply:
Changed to GetIncludeName()